docs/configuration.md

```diff
@@ -0,0 +1,89 @@
+# Configuration
+
+Configuring a repo to use `gha-scala-library-release-workflow` requires a `release.yml` GitHub workflow in your repo,
+and updated `sbt` settings.
+
+[Example GitHub pull requests](#examples) making these changes can be found further below.
+
+## GitHub workflow
+
+[Example `.github/workflows/release.yml`](https://github.com/guardian/etag-caching/blob/main/.github/workflows/release.yml)
+
+The functionality of `gha-scala-library-release-workflow` is provided in a
+[reusable workflow](https://docs.github.com/en/actions/using-workflows/reusing-workflows)
+called [`reusable-release.yml`](https://github.com/guardian/gha-scala-library-release-workflow/blob/main/.github/workflows/reusable-release.yml) -
+don't copy-and-paste that big file, instead just make a small `release.yml` workflow to
+[_call_](https://docs.github.com/en/actions/using-workflows/reusing-workflows#calling-a-reusable-workflow)
+it (as in the example file above).
+
+Your repo will require access to [release credentials](credentials.md) to
```

Comment:
Should we mention the repo has to be tagged with the `production` label?

Reply:
It's documented in https://github.com/guardian/github-secret-access (in fact, the CI tests on that repo fail and tell you if you forgot to do it!) but it does fit nicely into the new 'Repo settings' section, so yep!